meta-resin-common/recipes-containers/docker/docker_git.bb

```diff
@@ -59,7 +59,7 @@ do_compile() {
     x86_64)
       GOARCH=amd64
       ;;
-    i586)
+    i586|i686)
```

Comment:
Can you do as suggested here? https://github.com/mem/oe-meta-go/pull/4/commits/2585beb9660e72a3502b92d56a469b4c1d620e24